lib/modules/modules.go

```diff
@@ -174,6 +185,27 @@ func (f Features) ToProto() *proto.Features {
 	}
 }
 
+// ProductType is the type of product.
+type ProductType int32
+
+const (
+	ProductTypeUnknown ProductType = 0
+	// ProductTypeTeam is Teleport ProductTypeTeam product.
+	ProductTypeTeam ProductType = 1
+	// ProductTypeEUB is Teleport Enterprise Usage Based product.
+	ProductTypeEUB ProductType = 2
+)
+
+// IsLegacy describes legacy product that existed before EUB product
+// was introduced.
```

Comment:
```suggestion
// IsLegacy describes the legacy enterprise product that existed before the
// usage-based product was introduced. Some features (Device Trust, for example)
// require the IGS add-on in usage-based products but are included for legacy
// licenses.
```

Some additional context can't hurt.